app/css/description.css

```diff
@@ -0,0 +1,26 @@
+.grid__div-title__h1 {
```

Comment:
The [BEM](http://getbem.com/naming/) need to be descriptive, grid is the block div-title__h1 should be the element so only need one __ ( & need to be more descripted, not use html tags or css properties like grid to name)